apps/mapideas/templates/meinberlin_mapideas/mapidea_detail.html

```diff
@@ -69,8 +69,8 @@ <h1 class="resource-header__title">{{ object.name }}</h1>
                     style="height: 300px;"
                     data-map="display_point"
                     data-baseurl="{{ map_url }}"
-                    data-point="{{ object.point }}"
-                    data-polygon="{{ object.module.settings_instance.polygon }}"
+                    data-point="{{ point }}"
+                    data-polygon="{{ polygon }}"
```

Comment:
That's happening in core. https://github.com/liqd/adhocracy4/pull/57/commits/05cb62a7cd40cf4a540b1319a41ba1b16473220b